auth/ante.go

```diff
@@ -60,10 +66,19 @@ func NewAnteHandler(utxoMapper types.UTXOMapper, txIndex *uint16, feeAmount *uin
 		if !res.IsOK() {
 			return ctx, res, true
 		}
+
+		inputs := indenom1
+
 		// Verify the second input
 		if utils.ValidAddress(spendMsg.Owner2) {
+			addr2 := common.BytesToAddress(signerAddrs[1].Bytes())
```

Comment:
Why do we need to convert to bytes and back to address? Seems like addr2 := signerAddrs[1] should do exactly the same as the commented line

Reply:
signerAddrs[1] is of type sdk.Address

Reply:
yea you're right